src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -74,6 +80,8 @@ class LSFJobManager extends AbstractLSFJobManager {
         } else if (str.size() == "Jan 01 01:00 1000 L".size()) {
             // Again, strip away the status info.
             dateForParser = stripAwayStatusInfo(str)
+        } else {
+            throw new DateTimeException("The string ${str} is not a valid LSF datetime string and cannot be parsed.")
```

Comment:
if str is empty, the message becomes less readable. Use quotes `'${str}'`. Alsor a shorter message would be clearer: `Invalid datetime string: '${str}'`